Makefile

```diff
@@ -20,6 +20,7 @@ CITIES = Aachen Aarhus Adelaide Albuquerque Alexandria Amsterdam Antwerpen Arnhe
 				Zagreb Zuerich
 
 .DEFAULT_GOAL := help
+SHELL := /bin/bash
```

Comment:
Looks like you didn't get around to replacing all the /bin/sh with this variable.

Reply:
It changes it so that bash is used for all shell execution by default (look under the `set -e` lines to see how it allows us to use `$CID` without either `bash -c` or piping to files.

http://www.gnu.org/software/make/manual/make.html#Choosing-the-Shell

Reply:
Pushed another commit that does this now.